tls-client-with-opaque-keys/README.md

```diff
@@ -0,0 +1,114 @@
+# TLS Client with Opaque keys
+This sample demonstrates a TLS client sample application that uses a HW Crypto engine with Opaque keys feature. In this first version this sample is tied to specific device [ATCAECC508A](https://www.microchip.com/wwwproducts/en/ATECC508A). It's driver is developed inside [mbed-os](https://github.com/ARMmbed/mbed-os/pull/6104).
```

Comment:
The key thing about this sample is that it's a sample TLS client that performs client authentication.